src/Command/MarkMigrated.php

```diff
@@ -53,10 +53,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->setInput($input);
         $this->bootstrap($input, $output);
+        $Adapter = $this->getManager()->getEnvironment('default')->getAdapter();
```

Comment:
That's not the style we are following for the 3.0 releases, variables should be camelcased